annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastErrorReporter.java

```diff
@@ -82,4 +84,54 @@ private void reportErrorMessage(String message, Stream<Element> elements) {
         errorRaised = true;
         elements.forEach(element -> messager.printMessage(Diagnostic.Kind.ERROR, message, element));
     }
+
+    public void reportDuplicatePackagesIfSuchExist(Collection<FastPackageElement> fastPackages) {
+        HashMap<String, ArrayList<NamedFastPackage>> packagesByName = new HashMap<>();
+        AnonymousFastPackage anonymousFastPackage = null;
+
+        for (FastPackageElement fastPackage : fastPackages) {
```

Comment:
Collect all packages including anonymous in one place. Check if it's anonymous when you print message